src/picom.c

```diff
@@ -1445,15 +1519,54 @@ static void handle_present_events(session_t *ps) {
 			    ps->c, session_get_target_window(ps), 0, cne->msc + 1, 0, 0);
 			set_cant_fail_cookie(ps, cookie);
 
+			struct timespec now;
+			clock_gettime(CLOCK_MONOTONIC, &now);
+			uint64_t now_usec =
+			    (uint64_t)(now.tv_sec * 1000000 + now.tv_nsec / 1000);
+			uint64_t drift;
+			if (cne->ust > now_usec) {
+				drift = cne->ust - now_usec;
+			} else {
+				drift = now_usec - cne->ust;
+			}
```

Comment:
good ol' `abs`? or, if you feel *mysterious* today, https://graphics.stanford.edu/~seander/bithacks.html#IntegerAbs

Reply:
these 2 numbers are unsigned, so can't use `abs` here :smiling_face_with_tear: